### ISSUE TITLE
Pod install fail, RxSwift version conflicts

### ISSUE THREAD

**Issue by retsohuang** [AvdLee/Moya-SwiftyJSONMapper#4]
CocoaPods version 1.0.0

```
[!] Unable to satisfy the following requirements:

- `RxSwift` required by `Podfile`
- `RxSwift (~> 2.4)` required by `RxPermission (0.1.1)`
- `RxSwift (~> 2.0.0)` required by `Moya-SwiftyJSONMapper/RxSwift (1.0.3)`
- `RxSwift (~> 2.0)` required by `RxNimble (0.2.0)`
```

**Comment by liuduoios** [comment]
I have the same issue.

**Comment by AvdLee** [comment]
This should be fixed with the release of 1.0.4, please let me know if you still have any issues
